app/assets/src/api/index.js

```diff
@@ -262,6 +262,9 @@ const logAnalyticsEvent = (eventName, eventData = {}) => {
         czi_user: traits.biohub_user,
         demo_user: traits.demo_user,
         has_samples: traits.has_samples,
+        // For Google Analytics. See https://segment.com/docs/destinations/google-analytics/#track
+        label: JSON.stringify(eventData),
+        category: eventName.split("_")[0],
```

Comment:
Why category? This is the component name, right? 
Did we describe what category field should be for somewhere else?

Reply:
`label` and `category` are special keys for GA. I'll update the comment.